Rubberduck.Core/UI/Converters/ClassInstancingToBooleanConverter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+using Rubberduck.Refactorings.ExtractInterface;
+
+namespace Rubberduck.UI.Converters
+{
+    class ClassInstancingToBooleanConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
```

Comment:
Maybe I just didn't spot it, but I don't see this converter class actually used in your xaml file.  Maybe its use was abandoned during development?  If it is used and retained, I'm thinking a name like `PublicNotCreatableInstancingToBoolConverter` would be more consistent with the other `<something>ToBoolConverter` converters since PublicNotCreatable <==> true.